source/MissionAction.cpp

```diff
@@ -230,6 +230,21 @@ void MissionAction::Load(const DataNode &node, const string &missionName)
 			if(child.Size() >= 3)
 				paymentMultiplier += child.Value(2);
 		}
+		else if(key == "fine")
+		{
+			if(child.Size() < 2)
+				child.PrintTrace("Skipping invalid \"fine\" without amount entry:");
+			else
+			{
+				int64_t loadedFine = child.Value(1);
+				if(loadedFine > 0)
+					fine += loadedFine;
+				else
+					child.PrintTrace("Skipping invalid \"fine\" with non-positive value:");					
+			}
+			if(child.Size() > 2)
+				child.PrintTrace("Ignoring non-defined parameters after \"fine\" value entry:");				
```

Comment:
I don't believe we typically warn about excess parameters elsewhere.

Reply:
We might want to add more parameters in the future, so I removed the check.